apps/style/common.scss

```diff
@@ -1732,6 +1732,12 @@ a.WireframeButtons_active, div.WireframeButtons_active {
   margin-left: -400px;
 }
 
+html[dir='rtl'] .instructions-markdown img {
```

Comment:
basically same comment as charlie [here](https://github.com/code-dot-org/code-dot-org/pull/40034#pullrequestreview-634006868). i actually have a task to figure out how to _remove_ all usages of `!important` this sprint, so please don't add more! we can figure out how to apply these styles without it.

this would also add more stratification to the styles for this class, so let's try to clean that up as well. current style definitions for `.instructions-markdown`:
- [dashboard/app/assets/stylesheets/application.scss](https://github.com/code-dot-org/code-dot-org/blob/ecb26407e408fc605c0123abac321f8daa22d01f/dashboard/app/assets/stylesheets/application.scss#L2903) (there are multiple usages -- not just the one i linked to)
- [apps/style/studio/style.scss](https://github.com/code-dot-org/code-dot-org/blob/b598fa018617d95ea287a2496b3e07cb84fab6cc/apps/style/studio/style.scss#L140)
- [apps/style/craft/style.scss](https://github.com/code-dot-org/code-dot-org/blob/b598fa018617d95ea287a2496b3e07cb84fab6cc/apps/style/craft/style.scss#L1235) (styles for our minecraft tutorials -- shouldn't be relevant for what you're doing, but good to know about)

the selector for these styles is specific enough that i wouldn't think `!important` would be needed here at all -- is that not true?

Reply:
The purpose for using !important is to override inline styles that are present on the instructions markdown that is received from the back-end, which can only be done with !important. Alternatively, the inline styles could be removed from this markdown and we could apply non-important styles in the stylesheets you specified.

Reply:
ah, that makes sense. can you point me to where the inline styles are applied to the instructions markdown?

Reply:
is it just these 2 components? [<MarkdownInstructions/>](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/instructions/MarkdownInstructions.jsx) and [<LegacyDialogContents/>](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/ui/LegacyDialogContents.js)

Reply:
In the XML/Markdown that is entered into the level editor back-end. The image and the inline style we're trying to override is stored in the database.

Reply:
oh dear, okay. re: your earlier comment--

> Alternatively, the inline styles could be removed from this markdown and we could apply non-important styles in the stylesheets you specified.

is that a possibility? how much work does that entail? that seems like the best overall path because we shouldn't be applying styles in that way in the first place.

if it's easier, feel free to grab some time on my calendar and we can work through a solution (although i'll be OOO for a week starting this thursday).

Reply:
Yes, we need to set the float and padding for both RTL and LTR in this case, which is only about 5 more lines of CSS. 

I'm working with foundations to see how many markdown entries need to be updated. Hopefully not many.